js_modules/dagster-ui/packages/ui-core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -163,17 +163,17 @@ export const LaunchAssetExecutionButton = ({
   preferredJobName,
   additionalDropdownOptions,
   intent = 'primary',
-  showChangedAndMissingOption,
+  scopeAlwaysSingleAsset,
 }: {
   scope: AssetsInScope;
+  scopeAlwaysSingleAsset?: boolean;
   intent?: 'primary' | 'none';
   preferredJobName?: string;
   additionalDropdownOptions?: {
     label: string;
     icon?: JSX.Element;
     onClick: () => void;
   }[];
-  showChangedAndMissingOption?: boolean;
```

Comment:
`scopeAlwaysSingleAsset` doesn't roll off the tongue as smoothly. Maybe we could default the existing option to true and explicitly set it to false in the new call sites?

Reply:
That works for me!